api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,5 +117,31 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+
+	<changeSet id="20180214-1000" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="condition"/></not>
+		</preConditions>
+		<comment>
+			Add condition table
+		</comment>
+		<createTable tableName="condition">
+			<column name="condition_id" type="integer" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="additional_detail" type="varchar(255)">
+				<constraints nullable="true"/>
+			</column>
+			<column name="previous_version" type="org.openmrs.Condition"></column>
+			<column name="condition" type="varchar(50)" defaultValue="null"/>
+			<column name="clinical_status" type="enum('ACTIVE', 'INACTIVE', 'HISTORY_OF')" >
+				<constraints nullable="false" />
+			</column>
+			<column name="verification_status" type="enum('PROVISIONAL', 'CONFIRMED')" >
+				<constraints nullable="false"/>
+			</column>
+			<column name="onset_date" type="datetime" />
```

Comment:
Can you also add the following fields?
creator, date_created, voided, voided_by, void_reason

Reply:
Okay. I'll do that.

Reply:
Can you also add the uuid column?

Reply:
Done